src/app/layout.tsx

```diff
@@ -14,6 +14,8 @@ import { exposeFont, satoshiFont, tabularFont } from "../utils/fonts";
 import "./globals.css";
 config.autoAddCss = false;
 
+const trustedOrigins = `${process.env.NEXTAUTH_URL} https://api.portal.dev.gdi.lu`;
```

Comment:
important: as we discussed, it's better to introduce a new environment variable, we cannot hard code URL's in the code.